modules/accumulo/src/main/java/io/fluo/accumulo/iterators/GarbageCollectionIterator.java

```diff
@@ -87,7 +90,7 @@ public void init(SortedKeyValueIterator<Key, Value> source, Map<String, String>
       if (zookeepers == null) {
         throw new IllegalArgumentException("A configuration item for GC iterator was not set");
       }
-      oldestActiveTs = ZookeeperUtil.getOldestTimestamp(zookeepers);
+      oldestActiveTs = ZookeeperUtil.getGcTimestamp(zookeepers);
```

Comment:
Would changing this variable name to gcTimestamp make this code more clear?  I guess the gcTimestamp is technically the oldest active timestamp but it's a little confusing.